vendor/engines/secure_rooms/README.md

```diff
@@ -34,30 +34,19 @@ The door security hardware communicates through an API with the following endpoi
 
       ```
       { response: "deny",
-        reason:   "I only know how to deny right now." }
+        reason:   "No accounts found." }
       ```
 
-  In addition, sending identifiers that do not map to existing records will generate a not found:
-
-  **Not Found:** One of the input identifiers does not match an existing record.
-
-  * **Code:** 404 Not Found
-  * **Response:**
-
-      ```
-      { response: "deny",
-        reason:   <Error Message stating missing record> }
-      ```
-
-* **Planned Responses:**
-
-  **Access Granted Immediately:** We can infer the cardholder's payment method, so they are cleared immediately.
+  **Access Granted:** We can infer the cardholder's payment method, so they are cleared immediately.
 
   * **Code:** 200 OK
   * **Body:**
 
       ```
-      { response: "grant" }
+      { response: "grant",
```

Comment:
With a `grant`, I don't think I need these extra details (e.g. `name`, `tablet_identifier`, etc.). You can send them if you want, but I think I can ignore them!